modules/core/client/views/enroll.client.view.html

```diff
@@ -1,4 +1,5 @@
-<section data-ng-controller="EnrollController">
+<section data-ng-controller="EnrollController" id="content">
```

Comment:
!fix Instead of declaring this outside the content like this, it would be better to declare this inside of the side-bar directive!

Like so:

<side-bar></side-bar>

(Inside of side-bar.html)
<section data-ng-controller="EnrollController" id="content">
//side-bar.html goes here
</section>